worlds/sc2/Regions.py

```diff
@@ -375,15 +376,40 @@ def make_grid_connect_rule(
     return lambda state: state.has(f"Beat {missions[connected_coords].mission_name}", player)
 
 
+def make_dynamic_mission_order(
+    world: World,
+    mission_order_type: int
+) -> Tuple[Dict[SC2Campaign, Dict[str, MissionInfo]], int, str]:
```

Comment:
Incorrect return type here as well, it's just passing along the `Dict[SC2Campaign, List[FillMission]]` from the `make_*()` methods

Reply:
Fixed.